nixops/backends/__init__.py

```diff
@@ -237,7 +257,9 @@ def send_keys(self):
               )
             )
             os.remove(tmp)
-        self.run_command("touch /run/keys/done")
+        self.run_command("mkdir -m 0750 -p /run/keys && "
+                         "chown root:keys  /run/keys && "
```

Comment:
Why is it necessary?
mkdir won't change mode if the  path already exists.
I would expect everything to be idempotent.
/run/keys is actually created ny NixOS' initrd.